### ISSUE TITLE
Looking for a maintainer

### ISSUE THREAD

**Issue by vlastavesely** [ApiGen/ApiGen#1020]
There are many of us who would like to see version `5.0.0` of `ApiGen` to be released. Despite the fact that `5.0.0-RC5` has been released already, there is so much work to be done. Currently, there are some serious performance problems that must be resolved and some missing features that must be implemented, before we can release a final stable version.

This project, like every other open-source project, stands and falls with two things: collaboration and passion. It is really hard for an open source project to be successful when maintained by a single developer. This is true even more when the project is not a passion for its developers. If the project is not the most important thing in your development life, you should probably to give a chance to someone else.

When @TomasVotruba announced first release candidate, I started to help with implementing missing features and fixing issues I encountered with, in order to have `ApiGen` finished. But during the time, the performance issues have arisen that need deeper inspection and so much more work. I would like to have `ApiGen` finished to generate a sexy API reference for my project, but I cannot invest as much of effort as is needed here for I do have other projects that are priority in this time.

This is why we need your help. To be honest, without *your* help, this project will not be done. If it is important for you and you really want to have it finished, support it: fix an issue or implement some feature and send a PR. We will be happy to merge it.

But, what is most important, we are looking for a maintainer who would be able to put his or hers effort into massive optimization and making `ApiGen` competitive. Do you think that you right for the job, or do you know someone? Let us know!

**Comment by jordanbrauer** [comment]
Are you only looking for a sole-maintainer? I just came across this tool the other day and I'd love to be apart of this project, however I am not wanting to head one right away that I am unfamiliar with.

**Comment by vlastavesely** [comment]
Hi, any help would be appreciated. We do not expect that there would appear a new maintainer out of blue. Starting with a new and unfamiliar project is kind of hard so that is why I am here :) I have implemented a few features and fixed a few bugs so I have some knowledge of the code and therefore I can help you. And if we both would got stuck, I think that @TomasVotruba, who have implemented most things, would help also if it is needed.

Like I said, any help is appreciated here, even if it were just small baby-steps 👍

**Comment by TomasVotruba** [comment]
Exactly as @vlastavesely said.
 I'm here if you need me :+1:

**Comment by jordanbrauer** [comment]
Perfect 👍 I'd love to come on board then!

You mention the following in your OP,

>Despite the fact that `5.0.0-RC5` has been released already, there is so much work to be done. Currently, there are some serious performance problems that must be resolved and some missing features that must be implemented, before we can release a final stable version.

So ... I guess to start, I have some requests/recommendations (mostly devops & project mgmt stuff) that may help in organizing an actionable list of immediate tasks. Should I create new issue(s) for these?

**Comment by vlastavesely** [comment]
> Should I create new issue(s) for these?

Sure :)

**Comment by nelson6e65** [comment]
I had been following the development, for usage in my project. But, I'm actually not very familiar with the generation output templates. So, I don't know how to help with this. Performace issues are related to the creation of templates in html?

And... sorry for the off-topic, but...

**About theme...**
There is a way to generate a `.json` file(s) (or `.xml`) with the information needed (no generate the html) and use [Vue.js](https://vuejs.org) (a JavaScript framwork for reactive interfaces) with a pre-created interface layout just _fill_ with that data?

**Comment by jaapio** [comment]
Hi,

I have deep respect for the maintainers of this project. I do completely understand what you are going through as maintainers of this project. We have up's and down's in https://github.com/phpDocumentor/phpDocumentor as well. And maybe we can join our forces. Although this project seems to be unmainained for some time right now. Because some users of ApiGen are now using https://github.com/phpDocumentor/phpDocumentor I would like to see how we can help people migrate from ApiGen to phpDocumentor.

When ApiGen was started it probably had some intention to do a thing different from what phpDocumentor did by those days. Since we were around at that time as well. If we can add those missing features to phpDocumentor to improve that would be very nice. 

Thanks for being around! And thanks for all your other projects!

**Comment by JanTvrdik** [comment]
Solved by me being the new maintainer.
